packages/website/src/pages/SearchResultPage.tsx

```diff
@@ -31,9 +32,14 @@ const ResultListRenderer: FunctionComponent<ResultListProps> = (props) => {
                     <h2>Search Results:</h2>
                     <div className="body-l-book plasma-description">Patate King!</div>
                     <div className="tile-grid">
-                        {state.results.map((result: Result) =>
-                            result.title !== 'Plasma Design System' ? (
-                                <Tile title={result.title} href={result.clickUri} description={result.excerpt} />
+                        {state.results.map(({title, clickUri, excerpt, uniqueId}: Result) =>
+                            title !== 'Plasma Design System' ? (
```

Comment:
it has nothing to do with your PR but instead of forgetting I'm telling you about it here :D

https://github.com/coveo/plasma/blob/master/packages/website/webpack.config.js#L39

Reply:
i'll change it in my own pr, coming soon this morning :smile:

Reply:
BANCO :D